perfdash/config.go

```diff
@@ -225,6 +225,20 @@ var (
 					Parser:           parseSchedulingThroughputCL("load"),
 				},
 			},
+			"SchedulingLatencyLoad": []TestDescription{
```

Comment:
Can you change this to LoadSchedulingLatency for consistency (e.g. LoadRequestCount_PrometheusSimple)?